From 4c367c6de558f4d7496161ed2a226f8da0e69ba9 Mon Sep 17 00:00:00 2001
From: Luigi Leonardi <leonardi@redhat.com>
Date: Sun, 15 Jun 2025 17:20:44 +0200
Subject: [PATCH] fix path configurazione

---
 src/conf.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/conf.c b/src/conf.c
index 7be4288..fde252b 100644
--- a/src/conf.c
+++ b/src/conf.c
@@ -12,7 +12,7 @@
 
 static const char *CONF_PATH_CWD = "./ceda-cemu.ini";
 static const char *CONF_PATH_HOME =
-    "/.config/it.glgprograms.retrofficina/ceda-cemu.ini";
+    "/usr/share/ceda-cemu/ceda-cemu.ini";
 
 #define CONF_WORD_SIZE 8
 #define CONF_PATH_SIZE 512
@@ -150,7 +150,7 @@ void conf_init(void) {
     if (loaded_path == NULL) {
         const char *home = getenv("HOME");
         if (home) {
-            (void)snprintf(path, CONF_PATH_SIZE, "%s/%s", home, CONF_PATH_HOME);
+            (void)snprintf(path, CONF_PATH_SIZE, "%s", CONF_PATH_HOME);
             if (ini_parse(path, conf_handler, conf_tuples) >= 0)
                 loaded_path = path;
         }
-- 
2.49.0

